x-pack/plugins/maps/public/classes/styles/vector/symbol_utils.js

```diff
@@ -6,46 +6,77 @@
  */
 
 import React from 'react';
-import maki from '@elastic/maki';
 import xml2js from 'xml2js';
+import { Canvg } from 'canvg';
+import calcSDF from 'bitmap-sdf';
 import { parseXmlString } from '../../../../common/parse_xml_string';
 import { SymbolIcon } from './components/legend/symbol_icon';
 import { getIsDarkMode } from '../../../kibana_services';
+import { MAKI_ICONS } from './maki_icons';
```

Comment:
Each icon in `maki_icons` can have multiple properties like `svg` and `label` with the possibility of adding more such as `anchor` to remove hard-coding icon ids in kibana like in the `getMakiAnchor` function.